deepchem/models/torch_models/layers.py

```diff
@@ -4792,3 +4792,88 @@ def forward(self, dist: torch.Tensor) -> torch.Tensor:
         dist = dist.unsqueeze(-1) / self.cutoff
         output: torch.Tensor = self.envelope(dist) * (self.freq * dist).sin()
         return output
+
+
+class EncoderRNN(nn.Module):
+    """Encoder Layer for SeqToSeq Model.
```

Comment:
Need to add this to layers.rst and torch_layers.csv

Reply:
Done